src/granite/table.cr

```diff
@@ -1,8 +1,18 @@
+# Adds a :nodoc: to granite methods/constants if `DISABLE_GRANTE_DOCS` ENV var is true
+macro noDoc(stmt)
+  {% if env("DISABLE_GRANTE_DOCS") == "true" %}
+    # :nodoc:
+    {{stmt.id}}
+  {% else %}
+    {{stmt.id}}
+  {% end %}
+end
```

Comment:
Clever. I like that the logic is centralized.

`noDoc` feels like a command though, which would prevent anything from being documented. Also it's snakeCased, which doesn't feel very Crystal.

Naming isn't my forte, but is there another name that would work better here?

Reply:
Yea...i'll think of something better